cnames_active.js

```diff
@@ -3162,6 +3162,16 @@ var cnames_active = {
   "tori": "boostcode.github.io/torijs",
   "torino": "torinojs.github.io", // noCF? (don´t add this in a new PR)
   "torlondev": "torlondev.github.io",
+  "testing": {
+  "owner": {
+    "repo": "https://github.com/free12345678901/testing",
+    "email": "free12345543210987@gmail.com"
+  },
+  "record": {
+    "CNAME": "free12345678901.github.io"
+  },
+  "proxied": false
+  }
```

Comment:
```suggestion
  "testing": "free12345678901.github.io/testing", // noCF
```
You should be adding a single new line to this file, as demonstrated by the _many_ other entries that already exist...